docs/content/docs/connectors/table/kinesis.md

```diff
@@ -40,10 +40,40 @@ Dependencies
 The Kinesis connector is not part of the binary distribution.
 See how to link with it for cluster execution [here]({{< ref "docs/dev/configuration/overview" >}}).
 
+### Versioning
+
+There are two available Table API and SQL distributions for the Kinesis connector. 
+This has resulted from an ongoing migration from the deprecated `SourceFunction` and `SinkFunction` interfaces to the new `Source` and `Sink` interfaces.
+
+The Table API and SQL interfaces in Flink only allow one TableFactory for each connector identifier. The following list clarifies the underlying interface that is used depending on the distribution selected:
```

Comment:
Also, need to explain what NOT to do, i.e. adding both `flink-sql-connector-aws-kinesis-streams` and `flink-sql-connector-kinesis` dependencies, because it would cause a clash of connector names

Reply:
Done